src/commands/memes/marshaller.ts

```diff
@@ -0,0 +1,15 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const ADDON_URL = 'https://tenor.com/view/cute-girl-trying-to-warning-pilot-anime-meme-gif-21114064';
+
+export const addon: CommandDefinition = {
```

Comment:
```suggestion
export const marshaller: CommandDefinition = {
```
And suggest `MessageCommandDefinition`